src/webui/webui.pri

```diff
@@ -19,7 +19,7 @@ SOURCES += \
     $$PWD/qtorrentfilter.cpp \
     $$PWD/abstractwebapplication.cpp
 
-# QJson JSON parser/serializer for using with Qt4
-lessThan(QT_MAJOR_VERSION, 5): include(qjson/qjson.pri)
+# QJson JSON parser/serializer for using with Qt < 5.5
+lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 5): include(qjson/qjson.pri)
```

Comment:
The docs seems to indicate that `:` is a logical AND. Link: http://doc.qt.io/qt-4.8/qmake-advanced-usage.html